Makefile

```diff
@@ -114,18 +114,32 @@ ccflags-y += \
 	     -DCONFIG_NRF700X_ANT_GAIN_5G_BAND1=0 \
 	     -DCONFIG_NRF700X_ANT_GAIN_5G_BAND2=0 \
 	     -DCONFIG_NRF700X_ANT_GAIN_5G_BAND3=0 \
-	     -DCONFIG_NRF700X_BAND_2G_LOWER_EDGE_BACKOFF=0 \
-	     -DCONFIG_NRF700X_BAND_2G_UPPER_EDGE_BACKOFF=0 \
-	     -DCONFIG_NRF700X_BAND_UNII_1_LOWER_EDGE_BACKOFF=0 \
-	     -DCONFIG_NRF700X_BAND_UNII_1_UPPER_EDGE_BACKOFF=0 \
-	     -DCONFIG_NRF700X_BAND_UNII_2A_LOWER_EDGE_BACKOFF=0 \
-	     -DCONFIG_NRF700X_BAND_UNII_2A_UPPER_EDGE_BACKOFF=0 \
-	     -DCONFIG_NRF700X_BAND_UNII_2C_LOWER_EDGE_BACKOFF=0 \
-	     -DCONFIG_NRF700X_BAND_UNII_2C_UPPER_EDGE_BACKOFF=0 \
-	     -DCONFIG_NRF700X_BAND_UNII_3_LOWER_EDGE_BACKOFF=0 \
-	     -DCONFIG_NRF700X_BAND_UNII_3_UPPER_EDGE_BACKOFF=0 \
-	     -DCONFIG_NRF700X_BAND_UNII_4_LOWER_EDGE_BACKOFF=0 \
-	     -DCONFIG_NRF700X_BAND_UNII_4_UPPER_EDGE_BACKOFF=0 \
+	     -DCONFIG_NRF700X_BAND_2G_LOWER_EDGE_BACKOFF_DSSS=0 \
```

Comment:
Please add CONFIG_NRF700X_TCP_IP_CHECKSUM_OFFLOAD=y as by default for Zephyr this is enabled.